mono/sgen/sgen-workers.c

```diff
@@ -54,6 +54,7 @@ enum {
 typedef gint32 State;
 
 static SgenObjectOperations * volatile idle_func_object_ops;
+static SgenObjectOperations *idle_func_object_ops_par, *idle_func_object_ops_nopar;
```

Comment:
These ops are not changed during a worker phase. They are set only during start_all_workers, which is followed by a worker signal which I assume, is enough to make it visible for the other threads., given that the workers are coming alive from a wait.